### ISSUE TITLE
Any plays for the ES version?

### ISSUE THREAD

**Issue by vincentdchan** [quilljs/delta#82]
Hello quill team:

I am developing an editor with the delta library. It seems that the version on NPM is a common.js version. (The code are under `dist/` folder)

I need a ESModule version. So I [forked the repo](https://github.com/vincentdchan/delta-es).

I want es version published with the common.js version:
`/dist` for common.js version
`/es` for es version
 
If you agree with this, I can submit a PR.

**Comment by vincentdchan** [comment]
I would strongly recommend using an esm package because Delta is a quite pure package.

**Comment by luin** [comment]
Yeah I agree that ESM is the future. However, since this package runs on Node.js, I am not certain that it is currently mature enough to publish as a pure ESM package. Some people still hate dynamic imports for consuming ESM packages from CJS env.

**Comment by Exac** [comment]
@luin Let's bump up the major version and publish only ESM code then?
